### PR TITLE
Move blessed mergable PRs into new column, fix blessed PRs with partial approvals

### PR DESCRIPTION
**Pull request by jakebailey**
~The new test reveals that blessing also bypasses needing _all_ owners of modified packages to approve. Working on figuring that one out.~

### CODE REVIEW

File: packages/mergebot/src/basic.ts
Comment (sandersn) [inline packages/mergebot/src/basic.ts:16]: does the weekly maintainer have to know which (Blessed) column to move to?

**Review by sandersn — APPROVED** [review]
Code looks OK but I don't think I know how to *use* this feature.

File: packages/mergebot/src/basic.ts
Comment (jakebailey) [reply packages/mergebot/src/basic.ts:16]: No, they should only ever put something into the first one, since I removed the "straight to mergable" thing.

**Comment by jakebailey** [comment]
The PR's not yet complete; working on a fix for the blessed PRs overriding any other reviews thing.

File: packages/mergebot/src/compute-pr-actions.ts
Comment (jakebailey) [inline packages/mergebot/src/compute-pr-actions.ts:119]: Dropped some of these to make them less of a hazard for code below; a lot of code was incorrectly reading out reviews for one package and applying them to the whole.

**Comment by jakebailey** [comment]
> I don't think I know how to _use_ this feature.

The use is just: If there's a PR in "needs maintainer review" and you think it's not doing anything harmful such that an owner can make the final decision, move the PR to "Waiting for Code Reviews (blessed)".

If all authors approve, it will eventually move to "Waiting for Merge (Blessed)" and be eligible for a self merge. If someone at any point complains, it leaves the "blessed" part of the flow, dropping back into "Needs Author Action", or eventually back into the maintainer queue.

File: packages/mergebot/src/basic.ts
Comment (sandersn) [inline packages/mergebot/src/basic.ts:68]: stupid idea (because the code is already good enough): why not `export type ApproverKind = (typeof approverKindOrder)[number]` ?

but also: numeric enums give you all the features you need with `enum ApproverKind { None, Other, Owner, Maintainer }` to simplify `getMaxApproverKind`.

**Review by sandersn — APPROVED** [review]
Looks good. One very optional suggestion about enums.

File: packages/mergebot/src/basic.ts
Comment (jakebailey) [reply packages/mergebot/src/basic.ts:68]: I'll do the former one, but I was avoiding the latter because the ApproverKind is a part of baselines and will end up showing up in the dumped JSON as regular numbers.

**Pull request merged**
